src/Docker/Runner.php

```diff
@@ -334,9 +334,29 @@ public function run(array $jobDefinitions, $action, $mode, $jobId, UsageFileInte
                 ' (row ' . $counter . ' of ' . count($jobDefinitions) . ')'
             );
         }
+        $this->waitForStorageJobs($outputs);
         return $outputs;
     }
 
+    private function waitForStorageJobs(array $outputs)
+    {
+        $jobIds = [];
```

Comment:
nechybí tady check na featuru?

Reply:
tak potom tam jsou tyhle dvě zprávy úplně zbytečně. 

```
$this->loggersService->getLog()->info(sprintf('Found %s storage jobs, waiting for them to finish.', count($jobIds)));
$this->loggersService->getLog()->info('All storage jobs finished.');
```

Reply:
tu druhou jsem vyhodil, ta je tam opravdu naprd, ta prvni rika jestli tam teda neco je nebo neni, tak to mi neprijde moc zbytecne